h-youjin/LV.1/두-정수-사이의-합.js

```diff
@@ -0,0 +1,10 @@
+function solution(a, b) {
+  let answer = 0;
+  const maxNum = Math.max(a, b);
+  const minNum = Math.min(a, b);
+
+  for (let i = minNum; i <= maxNum; i++) {
+    answer += i;
+  }
+  return answer;
+}
```

Comment:
```suggestion
function solution(a, b){
    return (a+b)*(Math.abs(b-a)+1)/2; // 가우스 공식
}
```
Math.max와 Math.min을 활용하여 사잇값을 찾아주셨군요! 작성해드린 코드는 1부터 100까지 더하기 등에 활용되는 가우스 공식을 적용한 방식입니다. 저도 검색을 통해 알게 되었는데, 코드를 외워 두시면 다른 풀이에서도 도움을 받으실 수 있을겁니다! 잘 봤습니다! 🤗

Reply:
오 가우스 공식은 생각못했는데 ! 피드백 감사합니다 대운님! 😉